tests/visualization_tests/test_contour.py

```diff
@@ -1,10 +1,9 @@
+from __future__ import annotations
+
 from io import BytesIO
 import math
 from typing import Any
 from typing import Callable
-from typing import Dict
-from typing import List
-from typing import Optional
 from typing import Tuple
```

Comment:
I think it can be deleted.
```suggestion
```
In the case, line 33 should be changed as `RANGE_TYPE = Union[tuple[str, str], tuple[float, float]]`

Reply:
@eukaryo Unfortunately, it can't be changed in Python.

```
> git diff
diff --git a/tests/visualization_tests/test_contour.py b/tests/visualization_tests/test_contour.py
index 4bcfc6d46..644bf6885 100644
--- a/tests/visualization_tests/test_contour.py
+++ b/tests/visualization_tests/test_contour.py
@@ -31,7 +31,7 @@ from optuna.visualization.matplotlib._matplotlib_imports import Axes
 from optuna.visualization.matplotlib._matplotlib_imports import plt
 
 
-RANGE_TYPE = Union[Tuple[str, str], Tuple[float, float]]
+RANGE_TYPE = Tuple[str, str] | Tuple[float, float]
 
 parametrize_plot_contour = pytest.mark.parametrize(
     "plot_contour", [plotly_plot_contour, plt_plot_contour]

> python tests/visualization_tests/test_contour.py
Traceback (most recent call last):
  File "tests/visualization_tests/test_contour.py", line 34, in <module>
    RANGE_TYPE = Tuple[str, str] | Tuple[float, float]
TypeError: unsupported operand type(s) for |: '_GenericAlias' and '_GenericAlias'
```

Reply:
I confirmed that, `Union[tuple[str, str], tuple[float, float]]` is OK, not `tuple[str, str] | tuple[float, float]`.

Reply:
What is unsupported on < python3.10 is the `TypeAlias` feature, so wrapping types into `Union` goes well.
And you can use `RANGE_TYPE = "tuple[str, str] | tuple[float, float]"` for type alias on < python3.10.

Reply:
I think `RANGE_TYPE` does not use anywhere. Can't delete it?

Reply:
@gen740 Great catch! Removed it

Reply:
LGTM!